x-pack/test/functional/services/ml/data_frame_analytics_creation.ts

```diff
@@ -481,6 +525,11 @@ export function MachineLearningDataFrameAnalyticsCreationProvider(
       await testSubjects.existOrFail('mlAnalyticsCreateJobWizardCreateButton');
     },
 
+    async updateAnalyticsJob() {
+      await testSubjects.existOrFail('analyticsEditFlyoutUpdateButton');
+      await testSubjects.click('analyticsEditFlyoutUpdateButton');
```

Comment:
We should also validate that the button click did something, e.g. waiting for the flyout to close.